src/containerapp/azext_containerapp/tests/latest/test_containerapp_sessionpool.py

```diff
@@ -181,3 +181,68 @@ def test_containerapp_sessionpool_registry(self, resource_group):
         sessionpool_list = self.cmd("containerapp sessionpool list -g {}".format(resource_group)).get_output_in_json()
         self.assertTrue(len(sessionpool_list) == 0)
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer()
+    def test_containerapp_sessionpool_registry_identity(self, resource_group):
+        location = 'eastasia'
```

Comment:
if you want to hard code the region, there is no meaning to use configure command here. We have configure location command to make all tests run in same region. Why do you need to hard code to east asia?